legacy/emulator/strl.c

```diff
@@ -42,3 +42,11 @@ size_t strlcat(char *dst, const char *src, size_t size) {
 }
 
 #endif
+
+char *itoa(int i, char *dest, int base) {
```

Comment:
Please drop if not needed

Reply:
a64092b30dbf004a2820f29d8ff5687da0b8e0ab